cmd/kubectl-gadget/main.go

```diff
@@ -93,6 +93,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	if !skipInfo && !runtimeGlobalParams.Get(grpcruntime.ParamGadgetNamespace).IsSet() {
+		gadgetNamespace, found := utils.GetRunningGadgetNamespace()
+		if found {
+			runtimeGlobalParams.Set(grpcruntime.ParamGadgetNamespace, gadgetNamespace)
+		}
```

Comment:
A warning message is printed in `GetRunningGadgetNamespace()` itself when we find more than 1 running IG instance

Reply:
As I mentioned somewhere else, I think it's better to error out instead of printing something.

Reply:
Error is printed when there are multiple IGs running (Shouldn't happen) and when IG isn't running at all (except for `deploy`)

Reply:
I reverted the change.
For `deploy`,  `undeploy`, `version` and `--help` the namespace will not fail, since that should always work even if Inspektor Gadget is not deployed, right?